src/renderer/reducers/index.js

```diff
@@ -23,6 +25,7 @@ export type State = {
   settings: SettingsState,
   onboarding: OnboardingState,
   UI: UIState,
+  swap: SwapStateType,
```

Comment:
So this swap state is not stored on file, correct?
I say this because if it is, then we might as well just bring all the swap namespace from the settings state here too 🤔

Reply:
Yes, it is not stored on file.
The data is only stored in the top-level app's state, if you reboot the app or refresh it, the data stored here is no longer stored.

Reply:
Then it makes sense to keep it separate from the `settings.swap` namespace, gotcha